spec/requests/api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require "swagger_helper"
+require "rails_helper"
+
+RSpec.describe Api::V1::UsersController, swagger_doc: "v1/users_swagger.json", backend: true do
+  before(:all) do
```

Comment:
Issue: If tests fail or are interrupted, the directory won't be restored, potentially affecting other tests.
Recommendation: Use an around hook with ensure to guarantee cleanup:
```
around do |example|
  original_dir = Dir.pwd
  Dir.chdir Rails.root.join("swagger", "v1")
  example.run
ensure
  Dir.chdir original_dir
end
```